paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/DefaultFlowController.kt

```diff
@@ -129,7 +125,14 @@ internal class DefaultFlowController @Inject internal constructor(
      */
     lateinit var flowControllerComponent: FlowControllerComponent
 
-    private var paymentLauncher: StripePaymentLauncher? = null
+    private val intentConfirmationHandler = IntentConfirmationHandler.Factory(
+        intentConfirmationInterceptor = intentConfirmationInterceptor,
+        paymentConfigurationProvider = lazyPaymentConfiguration,
+        statusBarColor = statusBarColor,
+        savedStateHandle = viewModel.handle,
+        stripePaymentLauncherAssistedFactory = paymentLauncherFactory,
+        application = application,
+    ).create(viewModelScope)
```

Comment:
We should have this in the constructor of `FlowController` at some point. Got it working originally but required a lot of boilerplate DI injection that I don't think should be scoped in this PR.